docs/iris/src/whatsnew/2.0.rst

```diff
@@ -192,6 +192,18 @@ Incompatible Changes
 * If the `packing` argument to `iris.save` is a dictionary, an error is raised
   if it contains any keys other than 'dtype', 'scale_factor' and 'add_offset'.
 
+* The deprecated :mod:`iris.fileformats.grib` was removed. All Iris GRIB
+  functionality is now delivered through :mod:`iris-grib`.
+
+* In Iris v1 it was possible to configure Iris to log at import time through
+  :attr:`iris.config.LOGGING`. This capability has been removed in iris v2.
+
+* When coordinates have no well defined plot axis, iris.plot and iris.quickplot
```

Comment:
Links to modules?

Also, 'axis' --> 'axes', probably. Update: In fact, I think that either works here. Up to you whether you change it or not.